frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/document_cover_widget.dart

```diff
@@ -940,3 +1030,84 @@ class _DocumentIconState extends State<DocumentIcon> {
     return child;
   }
 }
+
+@visibleForTesting
+class AlignCoverButton extends StatelessWidget {
+  const AlignCoverButton({
+    super.key,
+    required this.onTap,
+  });
+
+  final VoidCallback onTap;
+
+  @override
+  Widget build(BuildContext context) {
+    final fillColor = UniversalPlatform.isDesktopOrWeb
+        ? Theme.of(context).colorScheme.surface.withValues(alpha: 0.5)
+        : Theme.of(context).colorScheme.onSurfaceVariant.withValues(alpha: 0.5);
+    final svgColor = UniversalPlatform.isDesktopOrWeb
+        ? Theme.of(context).colorScheme.tertiary
+        : Theme.of(context).colorScheme.onPrimary;
+    return FlowyIconButton(
+      hoverColor: Theme.of(context).colorScheme.surface,
+      fillColor: fillColor,
+      iconPadding: const EdgeInsets.all(5),
+      width: 28,
+      icon: FlowySvg(
+        FlowySvgs.table_align_center_s,
+        color: svgColor,
+      ),
+      onPressed: onTap,
+    );
```

Comment:
For the newly added UI code, we're no longer using `FlowyIconButton` or `FlowyTextButton`. Instead, you can refer to the `appflowy_ui` package and use `AFButton` and `AFTextButton`.